lib/cc/cli/init.rb

```diff
@@ -10,7 +10,7 @@ class Init < Command
       def run
         if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
           warn "Config file .codeclimate.yml already present.\nTry running 'validate-config' to check configuration."
-          create_default_configs
+          create_default_engine_configs
```

Comment:
>  If it does not exist when you run init, #create_codeclimate_yaml will write it, and again it will exist before #create_default_engine_configs is called.

Sorry, yeah, I didn't get that originally. Makes sense.